doc/api/url.md

```diff
@@ -317,6 +317,15 @@ console.log(myURL.port);
 myURL.port = 1e10;
 console.log(myURL.port);
 // Prints 1234
+
+// Out-of-range numbers, which are converted to exponential 
```

Comment:
This is especially confusing given the previous example which says unconditionally that out-of-range numbers are ignored. Are we sure this behavior isn't simply a bug that needs fixing, rather than a defined behavior that needs documenting? It's hard to imagine anyone actually intentionally using this behavior.

Reply:
I agree with both points. I didn't have time to go through the actual [spec](https://url.spec.whatwg.org), however I was told it is indeed the case in:
https://github.com/nodejs/node/issues/19595
https://github.com/whatwg/url/issues/377

Obviously adding a range-check and some tests is a very straightforward solution for the unexpected behavior, but I'm not at all sure this is not a spec bug. 

In the meantime though, the documentation should perhaps be changed as it does not reflect the actual behavior expressed in node.